docs/user/config/index.md

```diff
@@ -6,16 +6,28 @@ The later is preferred since it's more customizable, but it depends on your need
 
 ## Jest preset
 
+### The 3 presets
+
+`ts-jest` comes with 3 presets, allowing to cover most of project's base configuration:
+
+| Preset name | Description |
+|---|---|
+| `ts-jest/presets/default`<br>or `ts-jest` | `ts-jest` will take care of `.ts` and `.tsx` files only, leaving JavaScript files as-is. |
```

Comment:
`leaving JavaScript files as-is` -> `ignoring JavaScript files`

Reply:
I think if we say ignoring users will think js files can't be required, but they simply a not preprocessed. But if you think it's ok like that, I'll change for your version as I don't trust my English ;-)

Reply:
that's a good point. This comment should be ignored :+1: